.circleci/config.yml

```diff
@@ -16,6 +16,11 @@ jobs:
           command: dnf install -y epel-release
       - run:
           command: dnf install -y ansible python3 python3-pip
+      - run:
+          name: "Change format for ansible output"
+          command: |
+            echo '[defaults]' >> /etc/ansible/ansible.cfg
```

Comment:
No need to reconfigure ansible.cfg - we can set before calling ansbile-playbook
ANSIBLE_STDOUT_CALLBACK=yaml
ANSIBLE_LOAD_CALLBACK_PLUGINS=1